apps/www/src/components/plate-ui/playground-turn-into-dropdown-menu.tsx

```diff
@@ -109,20 +108,22 @@ const defaultItem = items.find((item) => item.value === ELEMENT_PARAGRAPH)!;
 
 export function PlaygroundTurnIntoDropdownMenu(props: DropdownMenuProps) {
   const value: string = useEditorSelector((editor) => {
-    if (!isSelectionExpanded(editor)) {
-      const entry = findNode<TElement>(editor!, {
-        match: (n) => isBlock(editor, n),
-      });
+    let commonSelection: string | undefined;
+    const codeBlockEntries = getNodeEntries(editor, {
+      match: (n) => isBlock(editor, n),
+    });
```

Comment:
Have you tested this for selections containing nested blocks, such as traditional (non-indent) lists?

Reply:
By non-indent lists you meant bulleted list and numbered list right if you are it is working on them

Reply:
Plate has two kinds of lists. Indent lists, like on the main playground, don't use nesting. Traditional lists, which are more accessible and follow the standard UL, OL and LI nesting structure, might pose an issue since your `getNodeEntries` query will return both the outer list element and all list item elements.

Reply:
I think passing `mode: 'highest'` to `getNodeEntries` might be a workable fix, but you would need to test this on https://platejs.org/docs/list.

Reply:
code is working on https://platejs.org/docs/list.

Hey @12joan thanks a lot for taking out time and PR review would keep you valuable comments in mind for my future pull requests